shuup/core/__init__.py

```diff
@@ -29,6 +29,7 @@ class ShuupCoreAppConfig(AppConfig):
             "shuup.core.order_creator:OrderSourceMethodsUnavailabilityReasonsValidator",
             "shuup.core.order_creator:OrderSourceSupplierValidator",
         ],
+        "internal_product_type_provider": ["shuup.core.models._products:SimpleSupplierInternalProductTypeProvider"],
```

Comment:
export `SimpleSupplierInternalProductTypeProvider` in `models.py` so you won't need to use `_products` here.

Reply:
Not possible as we can't load the all the models to load the `SimpleSupplierInternalProductTypeProvider`

Reply:
what do you mean? otherwise we should move this to some other place like shuup.core.enums or something